### PR TITLE
Enhance readability

### PR DESCRIPTION
**Pull request by Maghraby**

### CODE REVIEW

**Comment by Funfun** [comment]
outdated branch, please update with recent master.

**Comment by Maghraby** [comment]
@Funfun done

File: lib/messagebird/http_client.rb [hunk 1]
```diff
@@ -54,27 +54,17 @@ def request_block(method, path, params = {})
...
-      when :patch
-        request = Net::HTTP::Patch.new(uri.request_uri)
-      when :post
-        request = Net::HTTP::Post.new(uri.request_uri)
-      when :put
-        request = Net::HTTP::Put.new(uri.request_uri)
-      else
-        raise MethodNotAllowedException
-      end
+      raise MethodNotAllowedException unless [:delete, :get, :patch, :post].include?(method)
```
Comment (Funfun) [inline lib/messagebird/http_client.rb]: 1. the change is not correct. missing :put method
2. the array needs be extracted to constant, suggested name `ALLOWED_METHODS`

Comment (Funfun) [reply lib/messagebird/http_client.rb]: would be cool to have a simple unit test before making any change

**Comment by Funfun** [comment]
@Maghraby please fix rubocop issues as well.

File: lib/messagebird/http_client.rb (hunk 1 above)
Comment (Maghraby) [reply lib/messagebird/http_client.rb]: I will do it, I thought it already in the place

Comment (Funfun) [reply lib/messagebird/http_client.rb]: the tests did not fail in your first push while it had missing :put method, so obviously no tests capture introduce change

**Pull request merged**